### PR TITLE
Added connect-sqlite3 typings

### PR DESCRIPTION
**Pull request by oof2win2**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [X] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [X] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [X] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [X] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [X] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "dtslint/dt.json" }`, and no additional rules.
- [X] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Review by samarmohan — COMMENTED** [review]
Awesome! Thanks for your work.

**Review by samarmohan — APPROVED** [review]

File: types/connect-sqlite3/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
+// Type definitions for connect-sqlite3 0.9
+// Project: https://github.com/rawberg/connect-sqlite3 (Does not have to be to GitHub, but prefer linking to a source code repository rather than to a project website.)
+// Definitions by: oof2win2 <https://github.com/oof2win2>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as session from 'express-session';
```
[the remark this answers is not in the document]
Comment (peterblazejewicz) [reply types/connect-sqlite3/index.d.ts:6]: pick side that is expected to use this DT most, can be fine tuned later IMO.

[the remark this answers is not in the document]
Comment (oof2win2) [reply types/connect-sqlite3/index.d.ts]: Think that it is resolved now, with Express types being used as they will most likely be installed if this is used in context with Express - which it most likely will be.

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@oof2win2 thx!

**Comment by oof2win2** [comment]
Ready to merge

**Pull request merged**